web/src/helpers/scales.js

```diff
@@ -1,19 +1,16 @@
-// TODO: Merge themes and scales
+import { scaleLinear } from 'd3-scale';
+import { range } from 'lodash';
 
+// TODO: Merge themes and scales
 import { themes } from './themes';
 
-const d3 = Object.assign(
-  {},
-  require('d3-array'),
-  require('d3-interpolate'),
-  require('d3-scale'),
-  require('d3-scale-chromatic'),
+const uniformSpread = ({ from, to, count }) => (
+  range(count).map(index => (from + (to - from) * index / (count - 1)))
```

Comment:
you could consider using d3.range:
https://github.com/d3/d3-array#range

Reply:
Thanks for the tip, but that didn't make it much nicer due to rounding-off errors and neither D3 nor lodash range including the end value by default.

What I decided to do instead is just be simple and list the domain steps explicitly in https://github.com/tmrowco/electricitymap-contrib/pull/2299/commits/1a2d2acd0c0bc63cad57ff46445808adbcda42b3. I also noticed that _15 m/s_ is very low for an upper cap on wind speed (the marker would often go off the scale), so I doubled it to _30 m/s_, makes sense?

![image](https://user-images.githubusercontent.com/1216874/77437799-5d641580-6de5-11ea-9cc7-c24552695b03.png)

Reply:
hmm are you sure it still looks nice visually?

Reply:
It looks different but to me it doesn't look worse - at least values don't go off scale anymore :)

_before_
![image](https://user-images.githubusercontent.com/1216874/77466446-9662b100-6e0a-11ea-8db2-c3e3300ed730.png)

_after_
![image](https://user-images.githubusercontent.com/1216874/77466512-ad090800-6e0a-11ea-8ad5-a8b5fccf3824.png)

I think the real problem now is with stream lines density, i.e. it would be nice to address https://github.com/tmrowco/electricitymap-contrib/issues/1157.